src/common/ErrorPage/styled.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export const ErrorContainer = styled.div`
+    text-align: center;
+    background-color: ${({ theme }) => theme.colors.Whisper};
+    height: 100%;
+`;
+
+export const StyledImage = styled.img`
+    width: 100px;
+    margin-top: 10%;
+`;
+
+export const ErrorTitle = styled.h1`
+    font-size: 36px;
+    font-weight: 600;
+    line-height: 120%;
+    margin: 24px auto;
+`;
+
+export const ErrorBody = styled.div`
+    font-size: 22px;
+    font-weight: 500;
+    line-height: 130%;
+`;
+
+export const ErrorButton = styled.button`
+    background-color: ${({ theme }) => theme.colors.ScienceBlue};
+    color: ${({ theme }) => theme.colors.White};
+    padding: 16px 24px;
+    border-radius: 5px;
+    border: none;
+    margin: 24px auto;
+    font-size: 14px;
+    line-height: 19px;
```

Comment:
Ja bym tutaj dał wartość 136%, żeby trzymać się konwencji wyżej, gdzie również zapisałaś line-height w procentach ;)

Reply:
akurat teraz to nie uwaga a tylko spostrzeżenie, w projekcie line heighty są właśnie jakieś abstrakcyjne i tak naprawdę to niby zaleca się pisać w formacie np. "1.5" ale tutaj trzeba się domyślać co autor miał na myśli :D